hazelcast/src/test/java/com/hazelcast/test/HazelcastTestSupport.java

```diff
@@ -297,6 +298,10 @@ public static Address getAddress(HazelcastInstance hz) {
         return getClusterService(hz).getThisAddress();
     }
 
+    public static Address getAddress(HazelcastInstance hz, EndpointQualifier qualifier) {
+        return new Address(getClusterService(hz).getLocalMember().getSocketAddress(qualifier));
```

Comment:
minor: could also be `return getClusterService(hz).getLocalMember().getAddressMap().get(qualifier);`?

Reply:
Actually, no, because the `getSocketAddress` falls-back to member if there is no match in the address-map (i.e. single port mode)